test/unit/utM3DImportExport.cpp

```diff
@@ -50,35 +50,32 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 using namespace Assimp;
 
-class utM3DImportExport : public AbstractImportExportBase {
-public:
-	bool importerTest() override  {
-        Assimp::Importer importer;
-        const aiScene *scene = importer.ReadFile(ASSIMP_TEST_MODELS_DIR "/M3D/cube_normals.m3d", aiProcess_ValidateDataStructure);
+TEST(utM3DImportExport, import_cube_normals) {
+    Assimp::Importer importer;
+    const aiScene *scene = importer.ReadFile(ASSIMP_TEST_MODELS_DIR "/M3D/cube_normals.m3d", aiProcess_ValidateDataStructure);
 #ifndef ASSIMP_BUILD_NO_M3D_IMPORTER
-		return nullptr != scene;
+    ASSERT_NE(nullptr, scene);
 #else
-        return nullptr == scene;
+    ASSERT_EQ(nullptr, scene);
 #endif // ASSIMP_BUILD_NO_M3D_IMPORTER
-    }
-
-#ifndef ASSIMP_BUILD_NO_EXPORT
```

Comment:
What is the intention for rewriting the testcase, when the result is the same?

Reply:
I wanted to add another one, and I thought it would be better to follow the structure of another tests. But because the new test revealed another error, I'll revert the changes in this file.